primary/src/helper.rs

```diff
@@ -60,7 +73,7 @@ impl<PublicKey: VerifyingKey> Helper<PublicKey> {
                 // the data source (dictated by the digests parameter). The results
                 // will be emitted one by one to the consumer.
                 PrimaryMessage::CertificatesRequest(digests, origin) => {
-                    self.process_certificates(digests, origin, false).await;
+                    let _ = self.process_certificates(digests, origin, false).await;
```

Comment:
do we want to log when these errors occur instead of ignoring them?

Reply:
Actually the reason I introduced those errors is specifically to have a "transparent" way of capturing them instead of me having to introduce `error!` messages. For the `process_certificates` the macro I've used is the following:
```
#[instrument(level="debug", skip_all, fields(certificate_ids = ?digests, mode = batch_mode), err)]
```

in the end you'll see that the `err` has been included. For methods that return a `Result`, by including the attribute `err` on the `instrument` macro we are automatically capturing the returned error (see docs [here](https://docs.rs/tracing/latest/tracing/attr.instrument.html))

So based on the setup of the methods those errors will be captured and printed to the spans / logs as well.

To be honest, I believe that we want to use this pattern in various places so we do have a common nice way of reporting errors without having to introduce custom `err` messages and also ensure that errors will be captured on the span (later on distributed tracing will come up as failures which is something we want)

Reply:
Thanks for a more detailed explanation! I think that makes quite a bit of sense.